src/components/table-container-head/table-container-head.js

```diff
@@ -14,4 +15,12 @@ const TableContainerHead = ({ titles }) => {
   );
 };
 
+TableContainerHead.propTypes = {
+  titles: PropTypes.arrayOf(PropTypes.string.isRequired)
+};
+
+TableContainerHead.defaultProps = {
+  titles: ['']
```

Comment:
Can it have a default value empty array for `titles`?

Reply:
Yes! Redone.